### PR TITLE
pyploy&image

### PR DESCRIPTION
**Pull request by vishalvora**
Added image and pyplot widgets

```python
# pyplot
ds.subheader('PyPlot')
arr = np.random.normal(1, 1, size=100)
import matplotlib.pyplot as plt
fig, ax = plt.subplots()
ax.hist(arr, bins=20)
ds.pyplot(fig)
```

```python
# image
ds.subheader('Image')
image = Image.open('test_image.jpg')
ds.image(image)
# Date picker
dd_date = ds.date_input(value=datetime.date(2019, 7, 6))
ds.write('Selected_Date '  + str(dd_date))
```

### CODE REVIEW

File: lib/datastack/server/ds_class.py [hunk 1]
```diff
@@ -416,15 +416,30 @@ def query(self, data):
         return ''
     
     def image(self, data):
+        import io
```
Comment (mayurpokiya) [inline lib/datastack/server/ds_class.py]: can we move this import to top?

File: lib/datastack/server/ds_class.py [hunk 2]
```diff
@@ -416,15 +416,30 @@ def query(self, data):
...
             "type":"image",
             "prop":{
-                "data": 'data:image/png;base64, '  + data
+                "data": 'data:image/png;base64, '  + img_str.decode("utf-8") 
             }
         }
         self.append_block(block)
 
+    def pyplot(self, fig):
+        import io
```
Comment (mayurpokiya) [inline lib/datastack/server/ds_class.py]: repetitive import can be avoide

File: lib/test_app.py [hunk 3]
```diff
@@ -17,6 +19,18 @@ def load_main_page():
 # sildebar
 ds.sidebar().subheader('Pages')
 
+# pyplot
+ds.subheader('PyPlot')
+arr = np.random.normal(1, 1, size=100)
+import matplotlib.pyplot as plt
```
Comment (mayurpokiya) [inline lib/test_app.py:25]: can be move to top

**Review by mayurpokiya — APPROVED** [review]

**Review by Karan2311 — APPROVED** [review]

**Comment by vishalvora** [comment]
merged manually

**Pull request closed without merging**